### ISSUE TITLE
Synced views

### ISSUE THREAD

**Issue by Lenart12** [Endebert/squadmc#69]
I think it would be a really nice feature that would set the app apart from others if we had the option to create "multiplayer" rooms where multiple users could view or edit the same calculations in real time.

**Comment by ufonius** [comment]
Something I had in mind when I decided to contribute to this app. It's something I want to work on, but currently no idea if and when I will look into.

**Comment by Endebert** [comment]
@RasmusGodske I think it's a really cool feature that you implemented. Only downside I see is that performance took a big hit, especially when using it on my phone.

If you can find a way to resolve that, or have the performance impact only when the feature is active, it would be a great addition to the app.

**Comment by RasmusGodske** [comment]
@Endebert Yeah I totally agree, the performance has gone down by fare, which I also consider a problem. I am continually working on it to try improve it. However currently I am other problems as well, like the sub targets haven't been implemented yet.

**Comment by Endebert** [comment]
@RasmusGodske Sure, I'd take a look if I find the time

**Comment by WilliamVenner** [comment]
Could be done using P2P with WebRTC
